src/Compilers/CSharp/Test/Semantic/FlowAnalysis/TryLockUsingStatementTests.cs

```diff
@@ -256,7 +256,7 @@ public void TryMethod(out byte para)
             Assert.Equal(0, controlFlowAnalysisResults.ReturnStatements.Count());
             Assert.False(controlFlowAnalysisResults.EndPointIsReachable);
             Assert.Equal(null, GetSymbolNamesJoined(dataFlowAnalysisResults.VariablesDeclared));
-            Assert.Equal("by", GetSymbolNamesJoined(dataFlowAnalysisResults.AlwaysAssigned));
+            Assert.Equal(null, GetSymbolNamesJoined(dataFlowAnalysisResults.AlwaysAssigned));
```

Comment:
>Assert.Equal(null, GetSymbolNamesJoined(dataFlowAnalysisResults.AlwaysAssigned)); [](start = 12, length = 81)

Why is ```by``` is no longer always assigned? It looks like the very first statement in the ```try``` does that.

Reply:
Because control never leaves the try since finally leaves via an illegal branch. In the code after the try we will never see the results of the assignment.

However this can only happen if we have an error in the code, so "correctness" of that really does not matter all that much, so if previous behavior seems better, I can get it back.

---
In reply to: [151041220](https://github.com/dotnet/roslyn/pull/23172#discussion_r151041220) [](ancestors = 151041220)

Reply:
Will revert changes that cause this.

---
In reply to: [151229064](https://github.com/dotnet/roslyn/pull/23172#discussion_r151229064) [](ancestors = 151229064,151041220)